common/src/main/scala/tsec/common/package.scala

```diff
@@ -17,12 +18,31 @@ package object common {
     override def getMessage: String = cause
   }
 
-
   trait StringNewt {
     type I <: String
 
     val is: Is[I, String]
   }
+  // ByteBuffer capacity based on https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html
```

Comment:
Could this be mad into a typeclass with bitraversal to and from the type being encoded in its laws. This is quite lawless presently and doesn't have to be as this follows a standard codec pattern.

Reply:
Or atleast as a `ByteEncoder` which imlies the same behavior. I believe this forms a `Prism` so prism laws could apply, but I may be wrong.

Reply:
@ChristopherDavenport Happy Christmas and thank you for your help ;-)